core/runtime/src/main/java/io/quarkus/runtime/Quarkus.java

```diff
@@ -65,7 +65,7 @@ public static void run(Class<? extends QuarkusApplication> quarkusApplication, B
             //we already have an application, run it directly
             Class<? extends Application> appClass = (Class<? extends Application>) Class.forName(Application.APP_CLASS_NAME,
                     false, Thread.currentThread().getContextClassLoader());
-            Application application = appClass.newInstance();
+            Application application = appClass.getDeclaredConstructor().newInstance();
```

Comment:
Really, you need to refrain from including changes that have nothing to do with the issue you're fixing in a PR. It should be a different PR. You can create a PR with the various cleanups you made that is clearly identified as such.

Reply:
Updated to remove unrelated changes.